subset/switches/readme.md

```diff
@@ -70,7 +70,7 @@ Example of all necessary parameters in the system.conf related to physical switc
 
     # Define the usi url using your docker0's ip and port 5000, and re-run bin/setup_base if you're upgrading from versions before 1.9.0
     # Make sure docker's ip range doesn't conflict with that of the switch. Default docker0 ip is 172.17.0.1 
-    usi.url=172.17.0.1:5000
+    usi_setup.url=172.17.0.1:5000

```

Comment:
Is this step still necessary if it auto-defaults?   Maybe say only do this if it's not the default local docker0 interface?